.circleci/config.yml

```diff
@@ -139,6 +147,78 @@ jobs:
             node -e "require('google-proto-files').loadSync('schema/google/showcase/v1alpha3/messaging.proto');"
             node -e "require('google-proto-files').loadSync('schema/google/showcase/v1alpha3/testing.proto');"
 
+  regeneration-check:
+    docker:
+      - image: golang:1.12
+    working_directory: /go/src/github.com/googleapis/gapic-showcase
+    steps:
+      - checkout
+      - run:
+          name: Get proto dependencies
+          command: git submodule init && git submodule update
+      - run:
+          name: Install protoc
+          command: |
+            apt-get update && apt-get install -y unzip
+            curl -o ~/protoc3.zip -L https://github.com/google/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip
+            unzip ~/protoc3.zip -d ~/protoc3
+            mv ~/protoc3/bin/* /usr/local/bin/
+            mv ~/protoc3/include/* /usr/local/include/
+      - run:
+          name: Get generators
+          command: |
+              go get github.com/golang/protobuf/protoc-gen-go
+              go get github.com/googleapis/gapic-generator-go/cmd/protoc-gen-go_cli
+              go get github.com/googleapis/gapic-generator-go/cmd/protoc-gen-go_gapic
+      - run:
+          name: Compile protos
+          command: go run ./util/cmd/compile_protos
+      - run:
+          name: Check if compilation caused diff.
```

Comment:
So this PR would trigger a diff from its regeneration. Is that expected? There aren't any proto changes in this PR (but maybe you were saving the regeneration from some earlier changes to test this?)

Reply:
Yeah I think there were some earlier changes that I forgot to run the script for (good thing I'm automating it 😄)